.github/workflows/release_manager.yml

```diff
@@ -0,0 +1,71 @@
+name: Release Manager
+
+on:
+  push:
+    tags:
+      - "v[0-9]*"
+
+env:
+  SENTRY_DSN: https://533e56d1b2d64314bd6092a574e6d0f1@o74047.ingest.sentry.io/215496
+
+jobs:
+  release_manager:
+    name: Release Manager
+    runs-on: ${{ matrix.os }}
+    environment: Deploy Outline Manager
```

Comment:
Let's rename this to "Production Outline Manager" so it's reads better.

Reply:
Done in f5fd9232964bf0fedf9f333772ae7b86ff29f24c